src/app.js

```diff
@@ -1,21 +1,6 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const { moveFileIn } = require('./moveFileIn.js');
+const [sourcePath, destinationPath] = process.argv.slice(2);
```

Comment:
cool sollution, but i think its would be nice add command parametr to mark operation? like -mv